docs/what.rst

```diff
@@ -0,0 +1,23 @@
+=====================
+What is a Certificate?
+=====================
+
+An SSL *certificate* contains a public key and a private key that work together to enable
+a web browser to communicate with your server securely over an encrypted SSL (secure socket layer)
+connection. The certificate also identifies your website so that the browser (and the person behind it)
+can trust your identity. SSL certificates have mainly been used to provide secure,
```

Comment:
I'm not sure this description is a great fit for our DV certificates because "trust your identity" might have a lot of conceptual baggage that doesn't apply here. For example, if someone read this and then we asked "hey, if there's a certificate for greatregionalbank.com, does that mean we can trust that it's Great Regional Bank's web site?" or "hey, if there's a certificate for awesomesoftwaredownloads.net, does that mean we can trust that awesomesoftwaredownloads.net is a legitimate web site?".

I've struggled with this issue a lot because I think that certificates were *always* presented this way when they were first introduced to the web, yet in fact in our DV service, we almost don't want users to get *any* information from the certificate, we just want it to help the browser detect MITM attacks and refuse to connect. Beyond that, we add just about no security value, and may aspire to a future where the browser doesn't even show any distinctive UI for HTTPS connections, but shows a security warning for HTTP. In this case the certificate really doesn't aspire to communicate anything to the user in the course of a typical interaction, but it does still protect them by defending against surveillance and manipulation by the network intermediaries.